src/styles/stars.scss

```
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../images/star.svg) no-repeat center;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star + &__star {
+    margin-left: 4px;
+  }
```

```suggestion
  &__star:not(:last-child) {
    margin-left: 4px;
  }
```
 It is optional but that is more common